packages/grafana-ui/src/components/Combobox/OptionListItem.tsx

```diff
@@ -6,13 +8,14 @@ interface Props {
   label: string;
   description?: string;
   id: string;
+  isGroup?: boolean;
 }
 
-export const OptionListItem = ({ label, description, id }: Props) => {
+export const OptionListItem = ({ label, description, id, isGroup = false }: Props) => {
   const styles = useStyles2(getComboboxStyles);
   return (
-    <div className={styles.optionBody}>
-      <span className={styles.optionLabel} id={id}>
+    <div className={styles.optionBody} aria-disabled={isGroup}>
+      <span className={cx(styles.optionLabel, { [styles.optionLabelGroup]: isGroup })} id={id}>
```

Comment:
Do we want to add some aria stuff to also mention the group when on an item?

Reply:
We agreed on doing this in a separate PR so we don't block this one from merge